dace/codegen/targets/fpga.py

```diff
@@ -716,6 +716,7 @@ def make_parameters(self, sdfg, state, subgraphs):
             data_to_node.update(
                 {node.data: node
                  for node in subgraph.nodes() if isinstance(node, dace.sdfg.nodes.AccessNode)})
+            rtl_subgraph = any([isinstance(node, nodes.RTLTasklet) for node in subgraph.nodes()])
```

Comment:
At first glace this looks like a graph object, not a bool, better to rename:
```suggestion
            is_rtl_subgraph = any([isinstance(node, nodes.RTLTasklet) for node in subgraph.nodes()])
```

Reply:
This has been implemented.